src/inc/quic_platform_winuser.h

```diff
@@ -222,6 +222,12 @@ GetModuleHandleW(
 #define CXPLAT_PASSIVE_CODE() CXPLAT_DBG_ASSERT(CXPLAT_IRQL() == PASSIVE_LEVEL)
 #define CXPLAT_AT_DISPATCH() FALSE
 
+//
+// Enable Supported Tests
+//
+
+#define QUIC_TEST_CONNECTION_POOL 1
```

Comment:
It seems weird to enable tests based on a macro in a product header. Shouldn't we enable / disable the test based on the usual macros for platform and UM vs KM? (`_WIN32` I believe?)

Reply:
We introduce a separate macro so we're not dependent upon the platform-specific build macros in general. This is so the tests/feature can be enabled for new platforms as support is added, without having to find all the places where the test was depending on the platform-specific macro.

Reply:
I get the idea. I just find it strange to basically configure our tests from the product code:
- it takes more searching to know where the test run (the config will be distributed through potentially multiple files if we ever add it to other modes)
- test logic leaks in product logic

Do you think we could achieve something similar with a section in a test header that group those defines for all tests?
Something like:

```
#ifdef _WIN32
#define QUIC_TEST_CONNECTION_POOL 1
#endif

#ifdef LINUX
...
#endif

```

And that way we can just add more to that one place when other tests need the same kind of conditional enablement?

Reply:
Sure, I like that idea. I was looking for a place where put this define and dismayed to discover that we didn't already have a delineated place for it.

Reply:
Hang on, I think I can solve my problem without adding this new flag, simplifying things.